libgstc/libgstc.c

```diff
@@ -365,6 +365,38 @@ gstc_client_ping (GstClient * client)
   return gstc_cmd_send (client, request);
 }
 
+GstcStatus
+gstc_client_debug (GstClient * client, const char* threshold, const int colors)
+{
+  GstcStatus ret;
+
+  gstc_assert_and_ret_val (NULL != client, GSTC_NULL_ARGUMENT);
+  gstc_assert_and_ret_val (NULL != threshold, GSTC_NULL_ARGUMENT);
+
+  /* Enable debug of the pipeline */
+  ret = gstc_cmd_update (client, "/debug/enable", "true");
+  if (ret != GSTC_OK) {
+    return ret;
+  }
+  /* Set the level of debug */
+  ret = gstc_cmd_update (client, "/debug/threshold", threshold);
+
+  if (ret != GSTC_OK) {
+    return ret;
+  }
+  /* Enable the color in debug */
+  if (colors == 0)
```

Comment:
Even though it's valid syntax, it's always a good practice to use brackets in single line body `if`s